eahub/templates/eahub/profiles.html

```diff
@@ -52,6 +52,10 @@
 
 {% endblock %}
 
+{% block feedback %}
```

Comment:
I couldn't see the button. When I moved the feedback block into the above block I could see it